packages/react-sdk/src/remix.test.ts

```diff
@@ -0,0 +1,60 @@
+import { expect, test } from "@jest/globals";
+import { generateRemixParams, getRemixRoute } from "./remix";
+
+test("convert home page to remix route", () => {
+  expect(getRemixRoute("")).toEqual("_index.tsx");
```

Comment:
`getRemixRoute` imo `get` is bad name having that you transform one route format to another